Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs

```diff
@@ -469,7 +474,7 @@ public void AddBuff(Buff.BuffType buff)
 
         if (buff == Buff.BuffType.Bounce)
         {
-            QuadDamageTimer.Value = NetworkManager.ServerTime.TimeAsFloat + 10;
+            BounceTimer.Value = NetworkManager.ServerTime.TimeAsFloat + 10;
```

Comment:
Why are we changing the timer we used?

Reply:
Because the bufftype is Bounce so it should be the bounceTimer which should be increased.